### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - Text Block] - Screen Reader is not conveying the 'Name' of table under 'Text Block'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8360]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 22H2 (OS Build 25300.1000)
URL: https://adaptivecards.io/explorer/TextBlock.html
Browser: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Text Block' under 'Card Elements' and press enter key to select it.
3. Navigate to the Table present under the Text block and observe the issue.

**Actual Result:**
While navigating to the table present under 'Text block' using 'T' key, screen reader is not conveying the name of the table. It is announcing only description of the table.

**Expected Result:**
While navigating to the table using 'T' key, screen reader should convey the name of the Table and it should announce as 'Text Block' table then it should announce the description.

**Note:**
This issue is repro throughout the application for the tables.

**Observation:**
Same issue repro through the NVDA Screen Reader.

**User Impact:**
It will impact the screen reader user if the name of the table is not defined as they will face difficulty understanding if there are multiple tables on the page with same description.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

https://user-images.githubusercontent.com/95913748/224633462-bfcca258-4af3-47b9-b393-afb4707aae7b.mp4

![MAS4 1 2-Screen Reader is not conveying the 'Name' of table under 'Text Block'](https://user-images.githubusercontent.com/95913748/224633671-c82e2f33-2747-47a6-87b3-3134105aee01.png)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Note: We generate the table from markdown. There isn't a way (that I know of) to add an appropriate title via markdown. We will need to convert the table to a list to resolve the issue.

**Comment by vagpt** [comment]
This issue is not repro on the environment below, hence closing this issue.

Test Environment:
URL: https://adaptivecards.io/explorer/TextBlock.html
OS Version: 23H2(OS Build 25931.1000)
Browser Version: 118.0.2060.1
